service/lambda/runtime/event/apigatewayauthorizerevt/definition.go

```diff
@@ -0,0 +1,149 @@
+//
+// Copyright 2017 Alsanium, SAS. or its affiliates. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package apigatewayauthorizerevt
+
+import "encoding/json"
+
+// Identity provides identity information about the API caller.
+type Identity struct {
+	// The API owner key associated with the API.
+	APIKey string
+
+	// The source IP address of the TCP connection making the request to
+	// Amazon API Gateway.
+	SourceIP string
+}
+
+// RequestContext provides contextual information about an Amazon API Gateway
+// Proxy Custom Authorizer event.
+//
+type RequestContext struct {
```

Comment:
The parameter "RequestID" is missing.
```json
    "requestContext": {
        "path": "/request",
        "accountId": "123456789012",
        "resourceId": "05c7jb",
        "stage": "test",
        "requestId": "...",
        "identity": {
            "apiKey": "...",
            "sourceIp": "..."
        },
        "resourcePath": "/request",
        "httpMethod": "GET",
        "apiId": "s4x3opwd6i"
    }
```
Please add it with a short description.

cf. [doc](http://docs.aws.amazon.com/apigateway/latest/developerguide/use-custom-authorizer.html)